application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/DefaultNativeConnectionProvider.java

```diff
@@ -76,6 +76,7 @@ public DefaultNativeConnectionProvider(final Config config,
                 .withSslEngineFactory(sslEngineFactory)
                 .withMetricRegistry(metricRegistry)
                 .withMetricsEnabled(config.getStatistics().isEnabled())
+                .withMetricPrefix(config.getStatistics().getPrefix())
```

Comment:
Perhaps it could be good to refactor some of these into just one thing. So you are just passing statistics.

Reply:
Statistics is in application module, application depends on core. We can't use statistics in core because that will lead to circular dependency.

Reply:
It would still be good to pair metrics into it's own object since we are sending it into a Connection but I am fine without it.